presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/DistinctOutputQueryUtil.java

```diff
@@ -110,12 +110,6 @@ public Boolean visitIntersect(IntersectNode node, Void context)
             return true;
         }
 
-        @Override
```

Comment:
It would be nice to add a test (like `TestLogicalPlanner`) for this kind of queries where you pass SQL and see that this optimization was fired. Especially for case with plan like:
```
- Aggregate
        group by x
   - LeftJoin
       - Identity projection (x=x, y=y)
           - Aggregate
                group by x, y
       - ...
```